src/common/templates/includes/header-user.njk

```diff
@@ -15,8 +15,7 @@
       </a>
     </div>
     <div class="govuk-header__content">
-
-      <a href="/home" class="govuk-header__link govuk-header__link--service-name">{{__('title')}}</a>
+      <a href="/" class="govuk-header__link govuk-header__link--service-name">{{__('title')}}</a>
```

Comment:
Original story mentioned that clicking this was essentially a log out because you then had to go to the login page when you clicked "Start Now" on the home page.

This is now not the case if a session is still present, so this is a bug that doesn't apply anymore.

WAVE kicks up a fuss that there are two adjacent links going to the same place, hence the want to change this.